npm/pkg/controlplane/translation/translatePolicy.go

```diff
@@ -362,12 +362,8 @@ func translateRule(npmNetPol *policies.NPMNetworkPolicy, direction policies.Dire
 		// #2.1 Handle IPBlock and port if exist
 		if peer.IPBlock != nil {
 			if len(peer.IPBlock.CIDR) > 0 {
-				ipBlockIPSet, ipBlockSetInfo := ipBlockRule(npmNetPol.Name, npmNetPol.NameSpace, direction, matchType, ruleIndex, peer.IPBlock)
+				ipBlockIPSet, ipBlockSetInfo := ipBlockRule(npmNetPol.Name, npmNetPol.NameSpace, direction, matchType, ruleIndex, j, peer.IPBlock)
```

Comment:
nit and no strong opinion: `j` --> `peerIndex`

Reply:
Done